components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h

```diff
@@ -32,8 +32,39 @@
  floating.
 
  This design defaults to 5 lines minimum and does not expand on overflow by default.
+
+ Defaults:
+
+ Active Color - Blue A700
+
+ Border Stroke Color - Clear
+ Border Fill Color - Clear
+
+ Disabled Color = [UIColor lightGrayColor]
+
+ Error Color - Red A400
+
+ Floating Placeholder Color Active - Blue A700
+ Floating Placeholder Color Normal - Black, 54% opacity
+
+ Inline Placeholder Color - Black, 54% opacity
+
+ Leading Underline Label Text Color - Black, 54% opacity
+
+ Normal Color - Black, 54% opacity
+
+ Rounded Corners - All
+
+ Trailing Underline Label Text Color - Black, 54% opacity
+
+ Underline Color Normal - Black, 54% opacity
+
+ Underline Height Active - 2p
```

Comment:
Is "2p" the same as Material's "dp"?

Reply:
`p` is points like iOS points.

Reply:
Got it. I wasn't sure which unit it was describing. I've noticed Apple tends to spell out "points", while Material uses "dp" as their "device-independent pixel" unit on Android.